src/Components/Inspirations/wordsData.js

```diff
@@ -0,0 +1,24 @@
+const wordsData = [
+  "Moon",
+  "Reunion",
+  "Chills",
+  "Universe",
+  "Love",
+  "Jetset",
+  "Clues",
+  "Calmness",
+  "Earthrise",
+  "Crystals",
+  "Adventure",
+  "Wanderlust",
+  "Cinema",
+  "Action",
+  "Mystery",
+  "Blues",
+  "Nightly",
+  "Friends",
+  "Reflect",
+  "Sunshine"
+]
```

Comment:
We can probably have a group brainstorming session to add a bunch of new words to this list.

Reply:
Yes!